pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
now that it's in python-modules, you need to use `buildPythonPackage` otherwise importing from python will not work.

also you can/should use buildPythonPackage from callPackage arguments

Reply:
Fixed.